src/CheckIfDead.php

```diff
@@ -130,68 +131,8 @@ public function areLinksDead( $urls ) {
 			// Merge back results from full requests into our deadlinks array
 			$deadLinks = array_merge( $deadLinks, $results );
 		}
-		return $deadLinks;
-	}
 
-	/**
-	 * Perform a complete text request, not just for headers
-	 *
-	 * @param array $urls URLs we are checking
-	 * @return array with params 'error':curl error number and
-	 *   'result':true(dead)/false(alive) for each element
-	 */
-	protected function performFullRequest( $urls ) {
```

Comment:
Why are you moving these functions? 
I'd really like to only see relevant changes in this pull request. If you want to do formatting changes, they should be in a different PR.